mama.py

```diff
@@ -803,12 +806,11 @@ def main_func(argv: List[str]):
     np.seterrcall(numpy_err_handler)
 
     # Attempt to print package version info (pandas has a nice version info summary)
-    logging.debug("Printing Pandas' version summary:")
-    try:
-        logging.debug("\n%s", pd.show_versions())
-    except Exception as exc:  # pylint: disable=broad-except
-        logging.exception(exc)
-    logging.debug("\n")
+    if logging.root.level <= logging.DEBUG:
```

Comment:
Is it possible to be below `DEBUG`, given the flags we provide?

Reply:
Probably not, but in case we ever add any logging levels that are even more verbose (perhaps prohibitively so, but nevertheless a last resort in trying to tease out a problem), we'll certainly want to include this in that case, as well.

Reply:
Ok sounds good.